### PR TITLE
completed sql-select exercise

### PR DESCRIPTION
**Pull request by OSAdams**
Screenshot of sql-select directory

image

### CODE REVIEW

File: sql-select/expensive-films.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+select "filmId",
+       "title"
+ from "films"
+ where "replacementCost" = 29.99
```
Comment (thebearingedge) [inline sql-select/expensive-films.sql:4]: For this one the idea is to use `order by` so you don't have to hard-code the price.

File: sql-select/cheap-to-rent.sql [hunk 2]
```diff
@@ -0,0 +1,5 @@
+select "filmId",
+       "title"
+  from "films"
+ where "rentalRate" > 1
```
Comment (thebearingedge) [inline sql-select/cheap-to-rent.sql:4]: We want films that are less than one dollar to rent.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by OSAdams** [comment]
image

**Comment by OSAdams** [comment]
image

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**